src/components/stepforms/CustomSqlStepForm.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
+    <StepFormHeader
+      :title="title"
+      :stepName="editedStep.name"
+      :version="version"
+      :backendError="backendError"
+    />
+    <label>
+      Please write you SQL query by referring to the result of the previous step. Refer to the
+      previous step using the <strong>##PREVIOUS_STEP##</strong> keyword
+    </label>
+    <CodeEditorWidget
+      v-model="editedStep.query"
+      placeholder="Write your custom Sql here"
+      :errors="errors"
+      data-path=".query"
+    />
+    <StepFormButtonbar />
+  </div>
+</template>
+
+<script lang="ts">
+import Component from 'vue-class-component';
+import { Prop } from 'vue-property-decorator';
+
+import { CustomSqlStep, PipelineStepName } from '@/lib/steps';
+import { getTranslator } from '@/lib/translators';
+
+import BaseStepForm from './StepForm.vue';
+import CodeEditorWidget from './widgets/CodeEditorWidget.vue';
+
+@Component({
+  name: 'custom-sql-step-form',
+  components: { CodeEditorWidget },
+})
+export default class CustomSqlStepForm extends BaseStepForm<CustomSqlStep> {
+  stepname: PipelineStepName = 'customsql';
+
+  @Prop({
+    type: Object,
+    default: () => ({ name: 'customsql', query: '' }),
```

Comment:
To help the user.... we can add a placeholder here... something like :
```
SELECT * FROM  ##PREVIOUS_STEP##
```
Just for the user to see how it can be presented.

Reply:
yes, good idea.